data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -166,6 +168,12 @@ do
     scp -o PreferredAuthentications=publickey -o StrictHostKeyChecking=false -o UserKnownHostsFile=/dev/null -r -q "core@[${master}]:/tmp/artifacts-${GATHER_ID}/*" "${ARTIFACTS}/control-plane/${master}/"
 done
 
+cluster_bootstrap_gather
+if [ -d "${HOME}/artifacts-api-server-temp" ]
```

Comment:
This should actually check for the resources directory, otherwise we could get a failure on the `mv`.
```suggestion
if [ -d "${HOME}/artifacts-api-server-temp/resources" ]
```